dictdiffer/__init__.py

```diff
@@ -273,7 +273,9 @@ def check(key):
     return _diff_recursive(first, second, node)
 
 
-def patch(diff_result, destination, in_place=False):
+def patch(
+    diff_result, destination, in_place=False, *, allow_missing_keys=False
```

Comment:
why are you setting a limit on the number of positional args? Why can't `allow_missing_keys` be positional?

Reply:
I always found it confusing in APIs when boolean flags are passed positionally, so I'm a fan of explicitly making it a keyword argument. However, this is opinion-based and I will remove it.